src/layer/marker/Marker.Drag.js

```diff
@@ -58,10 +58,11 @@ L.Handler.MarkerDrag = L.Handler.extend({
 		}
 
 		marker._latlng = latlng;
+		e.latlng = latlng;
 
 		marker
-		    .fire('move', {latlng: latlng})
-		    .fire('drag');
+		    .fire('move', e)
```

Comment:
I originally done that, but then changed my mind, finding this version lighter, but I'll do that then :)